docs/utils/utils.rst

```diff
@@ -125,6 +124,17 @@ Array Utils
 .. doxygenfunction:: fiction::convert_array
 .. doxygenfunction:: fiction::convert_array_of_arrays
 
+Unit Utils
+-----------
+
+**Header:** ``fiction/utils/unit_utils.hpp``
+
```

Comment:
@marcelwa Do you have an idea how I add the content from ``units_utils.hpp``?

Reply:
For aliases, you can use `.. doxygentypedef::`. Not sure how to handle the `using ...` statements. Maybe you can find something here: https://breathe.readthedocs.io/en/latest/directives.html

Reply:
Then maybe just add a bit of text here mentioning that there are these operators exposed via this file.